pinery-lims/src/main/java/ca/on/oicr/pinery/lims/SampleAttribute.java

```diff
@@ -29,7 +29,7 @@
     SAMPLE_RECEIVE_DATE("geo_receive_date", "Receive Date"),
     EXTERNAL_NAME("geo_external_name", "External Name"),
     TUBE_ID("geo_tube_id", "Tube ID"),
-    TARGETED_RESEQUENCING("geo_targeted_resequencing", "Targeted Resequencing"),
+    TARGETED_RESEQUENCING("geo_targeted_resequencing", "Targeted Resequencing", "Targeted Sequencing"),
```

Comment:
'Targeted Sequencing' should be removed as the attribute name was corrected to 'Targeted Resequencing' for consistency.

Ideally, this enum would allow 1:1 mappings only. I see there is an inconsistency with 'template type'/'Template Type' as well?